enterprise-suite/values.yaml

```diff
@@ -3,8 +3,8 @@
 #
 # lightbend-docker-commercial-registry.bintray.io/enterprise-suite/es-console
 esConsoleVersion: v1.0.1
-# lightbend-docker-commercial-registry.bintray.io/enterprise-suite/es-monitor-api
-esMonitorVersion: v1.0.9
+# lightbend-docker-commercial-registry.bintray.io/enterprise-suite/console-api
```

Comment:
Should the version bump go in a separate PR or is that just a function of the name change?

Reply:
It's related to the name change, since that is the version that changes to console-api.